packages/EasyErrorHandler/src/Bridge/Bugsnag/BugsnagReporterProvider.php

```diff
@@ -54,7 +31,8 @@ public function getReporters(): iterable
             $this->bugsnag,
             $this->errorLogLevelResolver,
             $this->threshold,
-            $this->ignoredExceptions
+            $this->ignoredExceptions,
+            $this->ignoredExceptionsResolver,
```

Comment:
We are not using trailing comas on function arguments.

Reply:
Fixed